src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -1892,67 +1892,95 @@ namespace AirflowNetworkBalanceManager {
+            state.dataAirflowNetwork->AirflowNetworkSimu.Control = Alphas(2);
+            state.dataAirflowNetwork->AirflowNetworkSimu.WPCCntr = Alphas(3);
+            state.dataAirflowNetwork->AirflowNetworkSimu.HeightOption = Alphas(4);
+            state.dataAirflowNetwork->AirflowNetworkSimu.BldgType = Alphas(5);
+
+            // Retrieve flag allowing the support of zone equipment
+            state.dataAirflowNetwork->AirflowNetworkSimu.AllowSupportZoneEqp = false;
+            if (UtilityRoutines::SameString(Alphas(9), "Yes")) {
+                state.dataAirflowNetwork->AirflowNetworkSimu.AllowSupportZoneEqp = true;
+            }
+
+            // Find a flag for possible combination of vent and distribution system
+            {
+                auto const SELECT_CASE_var(UtilityRoutines::MakeUPPERCase(state.dataAirflowNetwork->AirflowNetworkSimu.Control));
+                if (SELECT_CASE_var == "NOMULTIZONEORDISTRIBUTION") {
+                    state.dataAirflowNetwork->SimulateAirflowNetwork = AirflowNetworkControlSimple;
+                    SimAirNetworkKey = "NoMultizoneOrDistribution";
+                } else if (SELECT_CASE_var == "MULTIZONEWITHOUTDISTRIBUTION") {
+                    state.dataAirflowNetwork->SimulateAirflowNetwork = AirflowNetworkControlMultizone;
+                    SimAirNetworkKey = "MultizoneWithoutDistribution";
+                } else if (SELECT_CASE_var == "MULTIZONEWITHDISTRIBUTIONONLYDURINGFANOPERATION") {
+                    state.dataAirflowNetwork->SimulateAirflowNetwork = AirflowNetworkControlSimpleADS;
+                    SimAirNetworkKey = "MultizoneWithDistributionOnlyDuringFanOperation";
+                } else if (SELECT_CASE_var == "MULTIZONEWITHDISTRIBUTION") {
+                    state.dataAirflowNetwork->SimulateAirflowNetwork = AirflowNetworkControlMultiADS;
+                    SimAirNetworkKey = "MultizoneWithDistribution";
+                } else { // Error
+                    ShowSevereError(state,
+                                    RoutineName + CurrentModuleObject + " object, The entered choice for " + cAlphaFields(2) + " is not valid = \"" +
+                                        state.dataAirflowNetwork->AirflowNetworkSimu.Control + "\"");
+                    ShowContinueError(state,
+                                      "Valid choices are \"NO MULTIZONE OR DISTRIBUTION\",\"MULTIZONE WITH DISTRIBUTION ONLY DURING FAN OPERATION\"");
+                    ShowContinueError(state, "\"MULTIZONE WITH DISTRIBUTION\", or \"MULTIZONE WITHOUT DISTRIBUTION\"");
+                    ShowContinueError(state,
+                                      "..specified in " + CurrentModuleObject + ' ' + cAlphaFields(1) + " = " +
+                                          state.dataAirflowNetwork->AirflowNetworkSimu.AirflowNetworkSimuName);
+                    ErrorsFound = true;
+                }
```

Comment:
If state.dataAirflowNetwork->AFNDefaultControlFlag = false, the original code section to read AirflowNetwork:SimulationControl object is executed.